CMakeLists.txt

```diff
@@ -795,7 +795,7 @@ target_link_libraries (seastar
     GnuTLS::gnutls
     StdAtomic::atomic
     lksctp-tools::lksctp-tools
-    rt::rt
+    $<$<AND:$<BOOL:UNIX>,$<NOT:$<BOOL:APPLE>>>:rt::rt>
```

Comment:
how come seastar can work without `timer_create()`  if it is built on Apple on UNIX ?

Reply:
@tchaikov thanks, good point! I investigated this and turns out, I don't have seastar compiled. It's OFF by default 
```
option(WITH_SEASTAR "Build seastar components")
```

Apparently, when I hit that linking issue I was still trying to compile with seastar ON, and later found out it wasn't working, but already had this change committed.
I'll now try to compile again, if it fails even with this fix - I'll close the PR.

Thanks again!